mpsl/CHANGELOG.rst

```diff
@@ -5,6 +5,22 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Added
```

Comment:
I will add a MPSL 0.5.0-2.prealpha header here and do some minor fixes to the changes section

Reply:
Do we need the version header? We are not using the version info anywhere else

Reply:
We don't need it, but this is tracked as the 2.prealpha internally, so that's all. For the next release we should be able to follow the ble_controller release model I think.